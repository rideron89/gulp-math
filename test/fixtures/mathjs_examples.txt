gulpmath(1.2 * (2 + 4.5));
gulpmath(2 foot to foot);
gulpmath(sin(45 deg) ^ 2);
gulpmath(9 / 3 + 2i);
gulpmath(det([-1, 2\; 3, 1]));