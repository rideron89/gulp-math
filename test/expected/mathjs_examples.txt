7.8
2 foot
0.5
3 + 2i
-7